<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormNewPlaybook</class>
 <widget class="QWidget" name="FormNewPlaybook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nouveau Playbook</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_playbookName">
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="playbook_name_field">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="placeholderText">
          <string>du playbook...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Chemin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pathField"/>
       </item>
       <item>
        <widget class="QPushButton" name="locationButton">
         <property name="text">
          <string>Parcourir...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_hosts">
         <property name="text">
          <string>Hosts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="user_name_field">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>user</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="become_user_box">
         <property name="text">
          <string>become_user</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dialog_button_box">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
